DEPENDENCIES

```diff
@@ -21,9 +21,12 @@ maven/mavencentral/com.google.guava/listenablefuture/9999.0-empty-to-avoid-confl
 maven/mavencentral/com.google.j2objc/j2objc-annotations/2.8, Apache-2.0, approved, clearlydefined
 maven/mavencentral/com.google.protobuf/protobuf-javalite/3.22.3, BSD-3-Clause, approved, clearlydefined
 maven/mavencentral/com.goterl/lazysodium-java/5.1.1, MPL-2.0, approved, #10952
+maven/mavencentral/com.goterl/resource-loader/2.0.1, , restricted, clearlydefined
```

Comment:
This is marked as restricted with no issue number. You need to open an IP issue ticket with the Dash tool. Do you have a committer or should I do it for you?

Reply:
somehow this dependency has disappeared from the list after update

Reply:
Right and I also don't see a reference in the pom.xml. I'll mark this as resolved.